lib/awful/titlebar.lua

```diff
@@ -84,11 +85,13 @@ local function new(c, args)
             client = c,
             position = position
         }
+        args.context = context
```

Comment:
Sorry, I was too slow at deleting my comment, but also too fast (I noticed that it is defined, but deleted the comment after you already replied) :-)